tests/grid/test_constructors.py

```diff
@@ -177,27 +177,85 @@ def test_network_from_file():
     assert mg.xy_of_reference == (12345, 678910)
 
 
+def test_framed_voronoi_edge_nodes():
```

Comment:
I added this test to check the positions of the edge nodes, which don't depend on a random seed.

Reply:
That's a good idea, thank you